docs/getting-started/install-configure.md

```diff
@@ -201,29 +201,31 @@ curl -sL https://raw.githubusercontent.com/crossplane/crossplane/master/install.
 </div>
 </div>
 
-## Select Provider
+## Select a Getting Started Configuration
+
+Now from your terminal, install and configure one of the example `Configuration`
+packages:
 
-Now from your terminal, install and configure a provider for Crossplane to use
-for infrastructure provisioning:
 <ul class="nav nav-tabs">
-<li class="active"><a href="#provider-tab-aws" data-toggle="tab">AWS</a></li>
-<li><a href="#provider-tab-gcp" data-toggle="tab">GCP</a></li>
-<li><a href="#provider-tab-azure" data-toggle="tab">Azure</a></li>
-<li><a href="#provider-tab-alibaba" data-toggle="tab">Alibaba</a></li>
+<li class="active"><a href="#aws-tab-1" data-toggle="tab">AWS (Default VPC)</a></li>
+<li><a href="#aws-new-tab-1" data-toggle="tab">AWS (New VPC)</a></li>
+<li><a href="#gcp-tab-1" data-toggle="tab">GCP</a></li>
+<li><a href="#azure-tab-1" data-toggle="tab">Azure</a></li>
+<li><a href="#alibaba-tab-1" data-toggle="tab">Alibaba</a></li>
 </ul>
 <br>
 <div class="tab-content">
-<div class="tab-pane fade in active" id="provider-tab-aws" markdown="1">
+<div class="tab-pane fade in active" id="aws-tab-1" markdown="1">
 
-### Install AWS Provider
+### Install Configuration Package
 
```

Comment:
Can we have a short description of what `Configuration` means and what it contains?

Reply:
This has been addressed by moving the paragraphs from https://github.com/crossplane/crossplane/pull/2166#discussion_r585092430 to here